### PR TITLE
Adds a medical swimsuit to the fitness locker, sprited by anonymous

### PR DESCRIPTION
**Pull request by Gef21**
title says it all really

🆑 
 * rscadd: Added a medical swimsuit to the fitness locker

### CODE REVIEW

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -707,3 +707,10 @@
 	item_state = "rottensuit"
 	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/clothing.dmi', "right_hand" = 'icons/mob/in-hand/right/clothing.dmi')
 	_color = "rottensuit"
+
+/obj/item/clothing/under/medicalswimsuit
```
Comment (PJB3005) [inline code/modules/clothing/under/miscellaneous.dm]: There already seems to be an `/obj/item/clothing/under/swimsuit`, could you change this to that?

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -707,3 +707,10 @@
...
+	name = "medical swimsuit"
+	desc = "A fancy swimsuit with a similar colour scheme to that of the medical department"
```
Comment (PJB3005) [inline code/modules/clothing/under/miscellaneous.dm]: Period here.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

File: code/modules/clothing/under/miscellaneous.dm (hunk 1 above)
Comment (Gef21) [reply code/modules/clothing/under/miscellaneous.dm]: done and tested

File: code/modules/clothing/under/miscellaneous.dm (hunk 2 above)
Comment (Gef21) [reply code/modules/clothing/under/miscellaneous.dm]: done

**Comment by Exxion** [comment]
Looks more like the CE hardsuit than medical

**Review by PJB3005 — APPROVED** [review]

**Comment by aceedex** [comment]
It's paramedic colors I think

**Comment by clusterfack** [comment]
You can directly include images into your posts by doing ! [] (link)

**Comment by PJB3005** [comment]
Without spaces of course.

**Comment by TheFriggenSawbones** [comment]
NO

**Comment by JustSumBody** [comment]
beach map when

**Comment by kol1th** [comment]
Delete this pr right now.

**Comment by SonixApache** [comment]
what the fuck is this

**Comment by Exxion** [comment]
There we go
I was wondering when the complaining would start

Surprised it took that long

**Comment by Shadowmech88** [comment]
Kind of strange how it came all at once within the span of half an hour.

**Comment by clusterfack** [comment]
It's almost as if there is some place where people congregate, and a group that uses in some meta fashion that acts in a uniform manner to influence public opinion.

**Comment by jakmak5** [comment]
waifupandering/10

**Comment by Probe1** [comment]
Change it to underwear and I might merge it.

**Comment by Gef21** [comment]
I have no idea how to do that and the emojicracy don't like this so I'm just gonna close it

**Pull request closed without merging**